quick-start.md

```diff
@@ -105,3 +100,98 @@ sudo su
+      - `APPSMITH_MAIL_USERNAME`: SMTP username
+      - `APPSMITH_MAIL_PASSWORD`: SMTP password
+    - Oauth Configuration:
+      - Google Oauth:
+        - `APPSMITH_OAUTH2_GOOGLE_CLIENT_ID`: Client ID provided by Google for OAuth2 login
+        - `APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET`: Client secret provided by Google for OAuth2 login
+      - Github Oauth:
+        - `APPSMITH_OAUTH2_GITHUB_CLIENT_ID`: Client ID provided by Github for OAuth2 login
+        - `APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET`: Client secret provided by Github for OAuth2 login
+    - `APPSMITH_GOOGLE_MAPS_API_KEY`: Google Maps API key which is required if you wish to leverage Google Maps widget. Read more at: https://docs.appsmith.com/third-party-services/google-maps
+    - `APPSMITH_DISABLE_TELEMETRY`: We want to be transparent and request that you share anonymous usage data with us. This data is purely statistical in nature and helps us understand your needs & provide better support to your self-hosted instance. You can read more about what information is collected in our documentation https://docs.appsmith.com/telemetry/telemetry
+
+    After Heroku finishes setting up the app, click “View” and your Appsmith should be up and running. You will be taken to the account creation page, where you can enter credentials to create an account and get started.
+
+
+{% hint style="warning" %}
+- We use Heroku Redis addon for caching which required your account to have billing information, but we use the free plan of this addon so it will charge you nothing. Please make sure your account already finish providing billing information.
+- You may need to wait 2 - 3 minutes before accessing the application.
+{% endhint %}
+
+### Custom domain
+To create your custom domain with your app, please follow these steps below:
+- Go to your app's settings tab
+![App setting](.gitbook/assets/heroku-app-settings.png)
+- Click `Add domain` button in Domains section
+![Add domain button](.gitbook/assets/heroku-add-domain-button.png)
+- Input your domain name & click `Next`. Heroku will provide you a DNS Target that you can map your domain with  
+![Add domain form](.gitbook/assets/heroku-add-domain-form.png)  
+![Finish](.gitbook/assets/heroku-finish.png)
+
+- Make sure that your DNS Record is update so that your custom domain will map to `DNS Target`
+
+- Once you finish, now you can access Appsmith from your custom domain
+
+{% hint style="warning" %}
+- Once you use a custom domain, You might want to setup SSL for your dyno. Please check the official document of Heroku [how to configure SSL](https://devcenter.heroku.com/articles/ssl)
+- Your dyno will need to be upgrade to at least `hobby` type to use this feature of heroku
+
+{% endhint %}
+
+### Re-Deploy your App using Heroku CLI
+To re-deploy your app (re-build & re-run), make sure you have Docker & Heroku CLI setup locally then follow steps below:
```

Comment:
Kind of confusing in the first glance that Docker is needed to redeploy using Heroku CLI